pyneuroml/utils/__init__.py

```diff
@@ -0,0 +1,87 @@
+import neuroml
+
+def extract_position_info(nml_model, verbose):
```

Comment:
Is this something users can use too? Otherwise we may want to make it private by naming it `_extract_position_info` (leading underscore?)

Also thinking if it should go into pyneuroml/plot/__init__.py instead since it's sort of specific to the plotting bits and not a general util? :thinking:

Reply:
I mainly pulled it out so it could be used in the 3D plot too to support networks also. It's based on what's in the povray export...

Yes, it's mainly used for plotting now, but could be used for other forms of analysis, as a general purpose way to get quick dicts of positions (and connections/inputs, etc eventually), so maybe leave here for now...

Reply:
Sounds good. Will document it at some point and try to add a test too.